README.md

```diff
@@ -1,11 +1,24 @@
-#Vote 2016
+# Vote 2016
 
-To get the SASS working, run `sass --watch assets/stylesheets/sass:assets/stylesheets/output` from the project directory.
+**This repository has been deprecated. Please use the [`18F/vote-2016`] [18f-fork]
+fork to follow development for this project or to file any issues. You must open
```

Comment:
I might change "You must open all PRs" with "Please open pull requests against the master ..." (to sound a bit friendlier and more conversational).